app/src/main/res/layout/fragment_example.xml

```diff
@@ -1,95 +1,100 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout
+<layout xmlns:tools="http://schemas.android.com/tools"
 	xmlns:android="http://schemas.android.com/apk/res/android"
 	xmlns:app="http://schemas.android.com/apk/res-auto">
 
 	<androidx.constraintlayout.widget.ConstraintLayout
 		android:layout_width="match_parent"
 		android:layout_height="match_parent"
+		android:background="#FFFFFF"
 		android:gravity="center"
-		android:orientation="vertical"
-		android:padding="@dimen/spacing_large">
+		android:orientation="vertical">
 
-		<TextView
-			android:id="@+id/title"
+		<ImageView
+			android:id="@+id/imageView7"
+			app:layout_constraintTop_toTopOf="parent"
```

Comment:
Move this line to the bottom of this component. As Lucas said, the order of these assignations should be `android` > `app` > `tools`